src/main.rs

```diff
@@ -48,40 +66,55 @@ fn main() {
         .setting(AppSettings::SubcommandRequired)
         .get_matches();
 
-    println!("Welcome to a round of 'battleship'");
+    ///////////////////////////////////////////////////////////////////////////////////////////////
+    ///                                         Start                                           ///
+    ///////////////////////////////////////////////////////////////////////////////////////////////
+    println!("{}", Yellow.paint("Welcome to a round of 'battleship'"));
```

Comment:
Good advice, it's clearly a better way to colorize the whole output message. Makes it easier to remove coloring.